tests/unit/conftest.py

```diff
@@ -7,7 +7,9 @@
 import pytest
 from pyramid import testing
 from pyramid.request import Request
+from webtest import TestApp as WebTestApp  # No pytest, this isn't a test class
```

Comment:
```suggestion
import webtest
```

...and use it as `webtest.TestApp`?

Reply:
Done!